TM1py/Services/CellService.py

```diff
@@ -156,6 +158,18 @@ def clear_spread(
 
         return self._post_against_cellset(cellset_id=cellset_id, payload=payload, delete_cellset=True, **kwargs)
 
+    def clear_with_mdx(self, cube: str, mdx: str, **kwargs) -> Tuple[bool, str, str]:
+        view_name = "".join(['}TM1py', str(uuid.uuid4())])
+        code = "".join([
+            f"ViewCreateByMdx('{cube}','{view_name}','{mdx}',1);",
+            f"ViewZeroOut('{cube}','{view_name}');"])
+        process = Process(name="")
+        process.prolog_procedure = code
+
+        from TM1py import ProcessService
+        process_service = ProcessService(self._rest)
+        return process_service.execute_process_with_return(process, **kwargs)
```

Comment:
I think in order to be compliant with the rest of TM1py it would be better to raise a `TM1pyException`, when the clear fails, instead of returning the result tuple.
@rclapp what do you think?